frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -415,71 +441,62 @@ class Chart extends Component<Props, State> {
-                  </span>
-                  <span className={styles.diffValue}>
-                    {formatNumber(difference, 2)}
-                    <span className={styles.diffUnit}>%</span>
-                  </span>
-                </>
-              ) : chartTotal === 0 ? null : (<Skeleton fontSize="1.125rem" minWidth="125px" />)}
-            </span>
-          </div>
-          <div className={styles.filters}>
-            <button
-              className={display === 'week' ? styles.filterActive : undefined}
-              disabled={disableFilters}
-              onClick={this.displayWeek}>
-              {t('chart.filter.week')}
-            </button>
-            <button
-              className={display === 'month' ? styles.filterActive : undefined}
-              disabled={disableFilters}
-              onClick={this.displayMonth}>
-              {t('chart.filter.month')}
-            </button>
-            <button
-              className={display === 'year' ? styles.filterActive : undefined}
-              disabled={disableFilters}
-              onClick={this.displayYear}>
-              {t('chart.filter.year')}
-            </button>
-            <button
-              className={display === 'all' ? styles.filterActive : undefined}
-              disabled={disableFilters}
-              onClick={this.displayAll}>
-              {t('chart.filter.all')}
-            </button>
+            {!showMobileTotalValue ?
+              <span className={!hasDifferenece ? '' : (
+                styles[difference < 0 ? 'down' : 'up']
+              )} title={diffSince}>
+                {hasDifferenece ? (
+                  <>
+                    <span className={styles.arrow}>
+                      {(difference < 0) ? (<ArrowUp />) : (<ArrowDown />)}
+                    </span>
+                    <span className={styles.diffValue}>
+                      {formatNumber(difference, 2)}
+                      <span className={styles.diffUnit}>%</span>
+                    </span>
+                  </>
+                ) : chartTotal === 0 ? null : (<Skeleton fontSize="1.125rem" minWidth="125px" />)}
+              </span>
+              :
+              <span className={styles.diffValue}>
+                {this.renderDate(toolTipTime * 1000)}
+              </span>
+            }
           </div>
+          {!isMobile && <Filters chartFiltersData={chartFiltersData} />}
         </header>
-        <div className={styles.chartCanvas} style={{ minHeight: `${this.height}px` }}>
+        <div className={styles.chartCanvas} style={{ minHeight: `${!isMobile ? this.height : this.mobileHeight}px` }}>
           {(!chartIsUpToDate || chartDataMissing) ? (
-            <div className={styles.chartUpdatingMessage} style={{ height: `${this.height}px` }}>
+            <div className={styles.chartUpdatingMessage} style={{ height: `${!isMobile ? this.height : this.mobileHeight}px` }}>
```

Comment:
also a bit hard to read and some duplication with line 497, please save the height in a const before the render